ops/ccs-ops-misc/cert-checker/README.md

```diff
@@ -0,0 +1,5 @@
+To Run Script
```

Comment:
Nitpicky, but this should be markdown formatted text. Try

```markdown
## To run this script:

```shell
chmod +x ./cert-checker
mv .env.sample .env
vi .env # (see ops for the correct settings)
source .env && ./cert-checker
# ``` <- add closing ticks
```

Reply:
done